proto/mcu_pb.proto

```diff
@@ -186,9 +187,9 @@ message ActiveLogEvents {
 
 // Power Management
 
-message BatteryPower {
+message PowerManagement {
```

Comment:
Maybe it would be better to name this `MCUPowerStatus` (in the future we'll also be adding a `GUIPowerStatus` message for the Raspberry Pi, which is powered by a separate battery and a separate copy of the LTC4015 chip).